megamek/src/megamek/common/MULParser.java

```diff
@@ -284,74 +286,65 @@ public MULParser(){
         pilots = new Vector<>();
     }
 
-    public MULParser(InputStream fin){
+    public MULParser(final File file, final @Nullable GameOptions options) throws Exception {
         this();
-        parse(fin);
-    }
 
-    public void parse(InputStream fin){
```

Comment:
We should still keep an InputStream ctor around.

Reply:
I'd rather keep the single version with a nullable, and just improve the documentation. IMO it makes it less likely someone will miss the options inclusion as that is the standard use case

Reply:
Wait, misread... We don't utilize an InputStream ctor, so why bother having one?

Reply:
Testing would use an InputStream.

Reply:
Added an input stream constructor for unit testing, with a note to say it was just for unit testing.